docs/ref/template-response.txt

```diff
@@ -355,3 +355,209 @@ with a simple template and a context containing a queryset::
 
     def blog_index(request):
         return TemplateResponse(request, 'entry_list.html', {'entries': Entry.objects.all()})
+
+StreamingTemplateResponse and SimpleStreamingTemplateResponse
+=============================================================
+
+.. versionadded:: 1.9
+
+:class:`~django.template.response.StreamingTemplateResponse` and
+:class:`~django.template.response.SimpleStreamingTemplateResponse` allow the same features as
+:class:`~django.template.response.TemplateResponse` and
+:class:`~django.template.response.SimpleTemplateResponse` but stream the template instead of rendering it.
+
+SimpleStreamingTemplateResponse objects
+---------------------------------------
+
+.. class:: SimpleStreamingTemplateResponse()
+
+Attributes
+~~~~~~~~~~
+
+.. attribute:: SimpleTemplateResponse.template_name
+
+    The name of the template to be rendered. Accepts a backend-dependent
+    template object (such as those returned by
+    :func:`~django.template.loader.get_template()`), the name of a template,
+    or a list of template names.
+
+    Example: ``['foo.html', 'path/to/bar.html']``
+
+    .. deprecated:: 1.8
```

Comment:
I don't think it makes sense to include annotations to "1.8" since this class is new in 1.9. I'd omit documenting any deprecated features and you don't need to mention that certain attributes were added in 1.8. It may be better to point to the documentation of the parent class rather than duplicate the documentation of the inherited attributes, etc.